cmd/dagger/cmd/do.go

```diff
@@ -157,6 +157,7 @@ func init() {
 	doCmd.Flags().StringArrayP("with", "w", []string{}, "")
 	doCmd.Flags().StringP("plan", "p", ".", "Path to plan (defaults to current directory)")
 	doCmd.Flags().Bool("no-cache", false, "Disable caching")
+	doCmd.Flags().String("experimental-platform", "", "Set target build platform (experimental)")
```

Comment:
Maybe add an example.
For instance : (`eg: linux/amd64`)

Reply:
since it's `experimental` I didn't want to bloat the help too much since this will probably be used mostly by "advanced" used that should have a good understanding about how it works.